policy.json

```diff
@@ -0,0 +1,7 @@
+{
+    "default": [
+        {
+            "type": "insecureAcceptAnything"
+        }
+    ]
+}
```

Comment:
I‘d rather not have two RPMs owning `policy.json` in the distribution at the same time… but I don’t have a strong opinion on which one should own it.